website/docs/developer/deployment-guide/creating-orca-glacier-bucket.md

```diff
@@ -192,11 +196,69 @@ aws sts get-caller-identity
 }
 ```
 
-Replace the value of `arn:aws:iam::909121343565:root` with the value of your
+Replace the number in `arn:aws:iam::909121343565:root` with the value of your
 account number.
 
 The Resource value is the bucket and bucket paths that the Cumulus application
-can access. Replace `arn:aws:s3:::sandbox-orca-glacier-archive` with the name
+can access. Replace `sandbox-orca-glacier-archive` with the name
 of the Orca archive bucket created in the previous section.
 
+##### Reports Bucket:
+
+The policy shown below can be used with some minor
+modifications, which will be detailed below.
 
+```json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "Cross Account Access",
+      "Effect": "Allow",
+      "Principal": {
+        "AWS": "arn:aws:iam::909121343565:root"
+      },
+      "Action": [
+        "s3:GetObject*",
+        "s3:GetBucket*",
+        "s3:ListBucket",
+        "s3:PutObject",
+        "s3:PutObjectAcl"
+      ],
+      "Resource": [
+        "arn:aws:s3:::sandbox-orca-glacier-archive",
+        "arn:aws:s3:::sandbox-orca-glacier-archive/*"
+      ]
+    },
+    {
+      "Sid": "Inventory-sandbox-orca-glacier-archive",
+      "Effect": "Allow",
+      "Principal": {
+        "Service": "s3.amazonaws.com"
+      },
+      "Action": "s3:PutObject",
+      "Resource": "arn:aws:s3:::sandbox-orca-glacier-archive/*",
+      "Condition": {
+        "StringEquals": {
+      	  "s3:x-amz-acl": "bucket-owner-full-control",
+      	  "aws:SourceAccount": "782417781503"
+        },
+      	"ArnLike": {
+      	  "aws:SourceArn": "arn:aws:s3:::sandbox-orca-glacier-archive"
+      	}
+      }
+    }
+  ]
+}
+```
+Replace `orca-primary` with the name of your orca bucket.
+
+The Principal value is the AWS root user for your Cumulus application that will
+access the ORCA archive bucket.
+
+Replace the number in `arn:aws:iam::909121343565:root` with the value of your
+account number.
```

Comment:
Possibly refer back to the ORCA Archive bucket where you determine the account number from the Cumulus OU to use.

Reply:
Added